app/views/user_mailer/in_person_deadline_passed.html.erb

```diff
@@ -1,11 +1,6 @@
 <p><%= t('user_mailer.in_person_deadline_passed.body.greeting') %></p>
 <p>
-  <%= t(
-        'user_mailer.in_person_deadline_passed.body.canceled',
-        app_name: APP_NAME,
```

Comment:
I removed these variables because they weren't being used.  The change did not affect the email's appearance.